### DISCUSSION TITLE
Feature Request: Show balance, meta data and media of tokens and NFTs both in transactions and in the wallet overview.

### DISCUSSION THREAD

**Discussion by LeoniePhiline** [Emurgo/yoroi-mobile#1870] [Ideas]
Hi!

I received some tokens and an NFT. They show great in pool.pm. They show mediocre in Daedalus (you can see the ascii value of the token type and the ticker and amount). In Yoroi they are only visible in transactions, and only the addr1 and ascii, and only after opening a disclosure UI component. 

As a user, I’d want to see artwork and all metadata in the transaction and account balance.

**Comment by stackchain** [comment]
@vsubhuman @Marta-Ryzhok
